doc/user-guide/ggplot.md

```diff
@@ -185,6 +185,12 @@ We can show a histogram of multiple columns by setting `x=['cut', 'color']`
 (ggplot("diamonds", aes(x=["cut", "color"])) + geom_histogram())
 ```
 
+Showing multiple columns also work with a mix of categorical and numerical columns
```

Comment:
We can also plot histograms for a combination of categorical and numerical columns.

Reply:
Just to confirm, is this suggesting changing the wordings?

Reply:
yes

Reply:
Just made the change